### PR TITLE
Trunk 248

### PR DESCRIPTION
**Pull request by jwnasambu**
## Description of what I changed
I replace the TODOs  with "junit test"

## Issue I worked on
https://issues.openmrs.org/browse/TRUNK-248?filter=-1

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you include a link to the ticket as mentioned here? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by jwnasambu** [comment]
@dkayiwa thanks for the correction. I have worked on it

File: api/src/test/java/org/openmrs/ObsTest.java [hunk 1]
```diff
@@ -1032,7 +1032,7 @@ public void newInstance_shouldCopyMostFields() throws Exception {
         obs.setDateVoided(new Date());                                
         obs.setVoidReason("Some reason");    
         obs.setValueComplex("SDIODF7980IDSISF");
-        obs.setComplexData(new ComplexData("Complex data", "Complex data".getBytes()))
```
Comment (dkayiwa) [inline api/src/test/java/org/openmrs/ObsTest.java]: Is this change part of the ticket?

Comment (jwnasambu) [reply api/src/test/java/org/openmrs/ObsTest.java]: Sorry for the delay but the changes are part of the ticket TODO as described TODO in this link https://github.com/openmrs/openmrs-core/pull/2811/commits/e0929d880cec681056b1b45a3b9683a766ffac5e#diff-30732ed0378eb70e0c9eb7625ab3af04L1030

**Comment by dkayiwa** [comment]
Can you squash the commits into one?

**Comment by jwnasambu** [comment]
I have managed to squash to one commit

    On Wednesday, February 6, 2019, 11:50:14 AM GMT+3, dkayiwa <notifications@github.com> wrote:  
 
 
Can you squash the commits into one?

**Comment by dkayiwa** [comment]
Thanks @jwnasambu 👍

**Pull request merged**